webapi/src/test/scala/org/knora/webapi/e2e/v2/OntologyV2R2RSpec.scala

```diff
@@ -171,13 +172,22 @@ class OntologyV2R2RSpec extends R2RSpec {
         }
 
         "serve a project-specific property as JSON-LD using the value object schema" in {
-            val classIri = URLEncoder.encode("http://api.knora.org/ontology/incunabula/v2#pubdate", "UTF-8")
+            val classIri = URLEncoder.encode("http://0.0.0.0:3333/ontology/incunabula/v2#pubdate", "UTF-8")
 
             Get(s"/v2/ontologies/properties/$classIri") ~> ontologiesPath ~> check {
                 val responseJson = AkkaHttpUtils.httpResponseToJson(response)
                 assert(responseJson == incunabulaWithValueObjectsPubDate)
             }
         }
 
+        "serve two project-specific classes and their properties as JSON-LD using the value object schema" in {
+            val pageIri = URLEncoder.encode("http://0.0.0.0:3333/ontology/incunabula/v2#page", "UTF-8")
+            val bookIri = URLEncoder.encode("http://0.0.0.0:3333/ontology/incunabula/v2#book", "UTF-8")
+
+            Get(s"/v2/ontologies/classes/$pageIri/$bookIri") ~> ontologiesPath ~> check {
+                val responseJson = AkkaHttpUtils.httpResponseToJson(response)
+                assert(responseJson == incunabulaPageAndBookWithValueObjects)
```

Comment:
I was thinking that we should have fully expanded Iris in the tests (i.e. after a JSON processor has processed them). 

We could have an extra tests for our expected prefixes, but I do not think that the prefixes should actually matter as long as the expanded Iri is correct.

Reply:
The prefixes matter, because that's one of the reasons we're using JSON-LD. Providing nice prefixes makes life easier for web developers. If the expected prefix is not used, that's a bug.

Writing a separate test that just checks for prefixes would be more work. Why do unnecessary work?

Reply:
Moreover, the Knora API server is not going to provide expanded IRIs. It always uses prefixes. So the only way to get expanded IRIs would be to expand them in the test. But that would be a test of the JSON-LD processor itself, not a test of the Knora API server.

Reply:
I think that whenever possible, a test should look at the actual output of the server, without transforming that output first into some other form.

Reply:
I agree that a test should be about what Knora produces (and we should not include third party software).

Still I think in the end what counts is the expanded Iri, not the prefix.

We can decide next week about how to proceed with the tests.

Reply:
The expanded IRI is what you want for the SALSAH 2, but I think a web developer making a project-specific custom GUI will be happy to have prefixes, so we have to make sure they work.